csrc/ir/nodes.cpp

```diff
@@ -217,7 +217,8 @@ std::vector<PolymorphicValue> IndexPutAccumulateOp::evaluate(
     const std::vector<PolymorphicValue>& inputs) const {
   return {at::index_put(
       /*self=*/inputs.at(0).as<at::Tensor>(),
-      /*indices=*/{inputs.at(1).as<at::Tensor>()},
+      // NOTE: aten API doesn't allow the broadcast dimension
+      /*indices=*/{inputs.at(1).as<at::Tensor>().squeeze(-1)},
```

Comment:
This will still work with previous pytorch versions (since we are only removing the broadcast dimension), correct?